onnx/backend/test/case/node/gru.py

```diff
@@ -0,0 +1,65 @@
+from __future__ import absolute_import
```

Comment:
Where do the numbers come from? What if someone's code failed this test? How can the author debug his code? He would wondering, why the result should be such numbers, not the others?

Reply:
Please take a look at the updated PR. I removed the hardcoded numbers and instead included the numpy implementations based on formulas provided in ONNX documentation.